python/src/com/jetbrains/python/psi/types/PyModuleType.java

```diff
@@ -132,6 +109,103 @@ else if (location != null) {
     return null;
   }
 
+  private static void processImplicitPackageMembers(@NotNull PsiFileSystemItem anchor,
+                                                    @Nullable PsiElement location,
+                                                    @Nullable PyImportedModule importedModule,
+                                                    @NotNull Predicate<String> filter,
+                                                    @NotNull Processor<List<? extends RatedResolveResult>> resultProcessor) {
+    final List<PyImportElement> importElements = new ArrayList<>();
+    final PyFile module = PyUtil.as(PyUtil.turnDirIntoInit(anchor), PyFile.class);
+
+    final PyImportElement origImportElement = importedModule != null ? importedModule.getImportElement() : null;
+    if (importedModule != null && (location == null || !inSameFile(location, importedModule))) {
+      if (origImportElement != null) {
+        importElements.add(origImportElement);
+      }
+    }
+    else if (location != null) {
+      final ScopeOwner owner = ScopeUtil.getScopeOwner(location);
+      if (owner != null) {
+        importElements.addAll(getVisibleImports(owner));
+      }
+
+      if (module != null) {
+        if (!inSameFile(location, module)) {
+          importElements.addAll(module.getImportTargets());
+        }
+        final List<PyFromImportStatement> imports = module.getFromImports();
+        for (PyFromImportStatement anImport : imports) {
+          Collections.addAll(importElements, anImport.getImportElements());
+        }
+      }
+    }
+    else {
+      return;
+    }
+
+    if (anchor.getVirtualFile() == null) {
+      return;
+    }
+    final PsiElement footHold = location != null ? location.getContainingFile() : module;
+
+    if (footHold == null) {
+      return;
+    }
+
+    final List<QualifiedName> myQnames = QualifiedNameFinder.findImportableQNames(footHold, anchor.getVirtualFile());
```

Comment:
Our spellchecker thinks that `Qnames` is a typo. Could you rename it to `myQNames` or `myQualifiedNames` here and `locationQnames` below as well? Then the module will become green.